openedx/features/content_type_gating/helpers.py

```diff
@@ -14,3 +18,38 @@
 }
 LIMITED_ACCESS = Group(CONTENT_TYPE_GATE_GROUP_IDS['limited_access'], 'Limited-access Users')
 FULL_ACCESS = Group(CONTENT_TYPE_GATE_GROUP_IDS['full_access'], 'Full-access Users')
+LOG = logging.getLogger(__name__)
+
+
+def not_correct_modes_for_fbe(course_key, enrollment=None, user=None):
+    # If CONTENT_TYPE_GATING is enabled use the following logic to determine whether
+    # enabled_for_enrollment should be false
+    if course_key:
```

Comment:
Nit: It might be clearer to write this as:
```
if course_key is None:
    return False
```

As written, a) you're only implicitly checking `None` on the input, b) relying on `None` being falsy in the output, and c) adding an extra layer of nesting depth for the expected case.

Reply:
Also, the method might clearer if you phrase it in the positive (`has_correct_modes_for_fbe`, and then use `not` at the callsites if it's needed.